cypress/integration/Feed_spec.js

```diff
@@ -0,0 +1,30 @@
+describe('Feed', () => {
+  beforeEach(() => {
+    cy.intercept("https://api.nytimes.com/svc/topstories/v2/home.json?api-key=GKUzDD1VY9ssjZ1AGusX3ci6AeoXCaSr", {fixture: 'nyt-article-fetch.json'})
+      .intercept('https://api.dandelion.eu/datatxt/sent/v1/?lang=en&text=A new biography by Debby Applegate recounts the story of Polly Adler, who arrived in America from Russia at 13 and became New Yorks most successful brothel owner, befriending mobsters, policemen, politicians and writers&token=beb0091844524790b7672a69bac06a2a', {fixture: 'manhattan-madam-sentiment.json'})
+      .intercept('https://api.dandelion.eu/datatxt/sent/v1/?lang=en&text=The magazines Ethicist columnist on responding to infidelity&token=beb0091844524790b7672a69bac06a2a', {fixture: 'affair-sentiment.json'})
+  })
+
+  it('Should show the header', () => {
+    cy.visit('/').get('.cy-sad-btn').click()
+      .get('.cy-header-text').contains('WellNews')
+  })
+
+  it('Should, when the happy button is clicked, show the articles sorted in ascending order by sentiment', () => {
+    cy.visit('/').get('.cy-happy-btn').click()
+      .get('.cy-article-card').first().contains('Should I Tell a Facebook Friend I Had an Affair With Her Partner?')
```

Comment:
I never know how much testing is enough and usually I check all of the pieces of the part I am testing. For example, whether the src of the image is the one that I am expecting as well as the title. But I am not sure if it is necessary... maybe something I will ask during our stand up today.